Restarting CI fixes these intermittent failures

There are a few of these that are fixed by restarting the CI builds.  Here is one:

```
--- FAIL: TestPipelineThatCrashes (70.07s)
    require.go:153: /home/travis/gopath/src/github.com/pachyderm/pachyderm/src/server/pachyderm_test.go:928
    require.go:155: No error is expected but got rpc error: code = 4 desc = context deadline exceeded
```

A few more:

```
--- FAIL: TestPipelineThatCrashes (70.08s)
	require.go:153: /home/travis/gopath/src/github.com/pachyderm/pachyderm/src/server/pachyderm_test.go:931
	require.go:155: No error is expected but got rpc error: code = 4 desc = context deadline exceeded
```

```
--- FAIL: TestFailedJobReadData (30.09s)
	require.go:153: /home/travis/gopath/src/github.com/pachyderm/pachyderm/src/server/pachyderm_test.go:2361
	require.go:155: No error is expected but got rpc error: code = 4 desc = context deadline exceeded
```

--- a/src/server/pachyderm_test.go
+++ b/src/server/pachyderm_test.go
@@ -922,7 +922,7 @@ func TestPipelineThatCrashes(t *testing.T) {
 		Job:        jobInfos[0].Job,
 		BlockState: true,
 	}
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*60)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
 	defer cancel()
 	jobInfo, err := c.PpsAPIClient.InspectJob(ctx, inspectJobRequest)
 	require.NoError(t, err)
@@ -2352,7 +2352,7 @@ func TestFailedJobReadData(t *testing.T) {
 		BlockState: true,
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*60)
 	defer cancel() //cleanup resources
 	jobInfo, err := c.PpsAPIClient.InspectJob(ctx, inspectJobRequest)
 	require.NoError(t, err)

